src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3092,7 +3092,7 @@ public static void DCS_MyPersonSurrogate()
     [Fact]
     public static void DCS_FileStreamSurrogate()
     {
-        const string TestFileName = "Test.txt";
+        string TestFileName = Path.GetTempFileName();
```

Comment:
That's a good questions @danmosemsft. I guess it's because I didn't know it existed! Expect a modified PR soon.